en/quickstart/pywps_quickstart.rst

```diff
@@ -1,17 +1,17 @@
 :Author: OSGeo-Live
-:Author: Jachym Cepicky
-:Version: osgeo-live8.0-draft
-:License: Creative Commons Attribution-ShareAlike 3.0 Unported  (CC BY-SA 3.0)
+:Author: Jachym Cepicky, Tom Kralidis
+:Version: osgeo-live10.0
```

Comment:
Has to be marked as -draft until someone does a full review from within the disk